src/App.jsx

```diff
@@ -10,23 +10,26 @@ function App() {
   return (
     <div className="App">
       <nav className="navBar">
-        <Link className="bookstoreTitle" to="/BookStoreReact_Vite/">
-          Geeky Book Store
-        </Link>
-        <div className="navLinks">
-          <Link className="link" to="/BookStoreReact_Vite/books">
-            Books
-          </Link>
-          <Link className="link" to="/BookStoreReact_Vite/categories">
-            Categories
-          </Link>
-          <Link className="link" to="/BookStoreReact_Vite/contact">
-            Contact
-          </Link>
+        <div className="fstdiv">
           <Link className="bookstoreTitle" to="/BookStoreReact_Vite/">
-            <img className="accountPic" src={accountPic} alt="" />
+            Bookstore CMS
           </Link>
+          {/* <div className="navLinks"> */}
+            <Link className="link" to="/BookStoreReact_Vite/books">
+              Books
+            </Link>
+            <Link className="link" to="/BookStoreReact_Vite/categories">
+              Categories
+            </Link>
+            <Link className="link" to="/BookStoreReact_Vite/contact">
+              Contact
+            </Link>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirements, as your previous code reviewer also mentioned that your design should match **EXACTLY** with *Zeplin* design. It seems like your design still have some issues, like your navigation is not matching with the given design. Kindly ensure your navigation matches the given design
   - Kindly use the correct font family, you are using `Times New Roman`, instead of the given `Montserrat`
   - The colours of the `Books` and `Categories` links do not match the given design


### Your design 

![image](https://github.com/GeekyHacks/BookStoreReact_Vite/assets/66476329/756a5263-32a6-4476-9ffb-9767c9e94d3e)

### Given design 

![image](https://user-images.githubusercontent.com/81932478/265206087-12abe8d4-82ba-4945-a589-ee7a375ef452.png)